payloads/external/Makefile.mk

```
@@ -290,7 +290,8 @@ $(obj)/UEFIPAYLOAD.fd: $(DOTCONFIG) $(IPXE_EFI)
 		CONFIG_EDK2_DGPU_POWER_OPTION=$(CONFIG_EDK2_DGPU_POWER_OPTION) \
 		CONFIG_DGPU_ONLY_AVAILABLE=$(CONFIG_DGPU_ONLY_AVAILABLE) \
 		CONFIG_EDK2_FTDI_USB_UART_SUPPORT=$(CONFIG_EDK2_FTDI_USB_UART_SUPPORT) \
-		CONFIG_QEMU_PLATFORM=$(CONFIG_CPU_QEMU_X86)
+		CONFIG_QEMU_PLATFORM=$(CONFIG_CPU_QEMU_X86)\
```

nit:
```suggestion
		CONFIG_QEMU_PLATFORM=$(CONFIG_CPU_QEMU_X86) \
```
It's unusual to not have a space there.

---

it seemed a bit off, now i know what was wrong: https://github.com/Dasharo/coreboot/pull/696/commits/44852e97f1f4af7d389f9b170398c6e6680fdcbd